pkg/configuration/base/resources/pod.go

```diff
@@ -64,7 +64,7 @@ const (
 	JenkinsSCConfigReqMethod  = "POST"
 	JenkinsSCConfigReqRetry   = "10"
 	JenkinsSCConfigLabel      = "type"
-	JenkinsSCConfigLabelValue = "%s-jenkins-config"
+	JenkinsSCConfigLabelValue = "jenkins-config"
```

Comment:
why removing that?
in case of multiple jenkins instances being installed in a project, user wouldn't be able to specify which SCC to use.